react-app/webpack/webpack.dev.js

```diff
@@ -0,0 +1,7 @@
+const {merge} = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
```

Comment:
You can avoid duplicating `merge`-codeblocks within both `dev` and `prod` subconfigs. You may use that within `webpack.config.js` and pass current value of `--mode`-flag to the [function](https://webpack.js.org/configuration/configuration-types/#exporting-a-function) that would merge respective subconfig with the base one.